src/dji_osdk_ros/dji_vehicle_node.cpp

```diff
@@ -392,6 +396,7 @@ bool VehicleNode::initTopic()
   time_sync_pps_source_publisher_ = nh_.advertise<std_msgs::String>("dji_osdk_ros/time_sync_pps_source", 10);
 
   #ifdef ADVANCED_SENSING
+  rtsp_streamer_.on_configure_writer(1920, 1080);
```

Comment:
Here we configure the rtsp streamer server with a specific resolution, right? Could we do this in dynamic reconfiguration? Maybe you can delegate this to another developer and follow it up

Reply:
I'll create an issue for it, its a really good idea :fire: